questionnaire-zh-tw.json

```diff
@@ -0,0 +1,434 @@
+{
+  "specVersion": "2.1.1",
+  "language": "zh-tw",
+  "title": "一致性問題",
+  "contextText": "背景",
+  "questionsText": "問題",
+  "completedText": "已完成?",
+  "statusText": "狀態",
+  "answerColumnText": "回答",
+  "sectionColumnText": "部份",
+  "numberColumnText": "編號",
+  "specRefColumnText": "規格參考",
+  "questionColumnText": "問題集",
+  "tocText": "目錄",
+  "preambleText": "OpenChain工作項目通過簡化許可開源的合規性和保持一致性，建立起對於開放源的信任。為維護OpenChain規範 ，在定義核心數據方面必需満足合規程序的各項品質要求。其結果是，對於軟體供應鍵參與者而言，開源許可合規性變得更加可預測、可理解和高效率。\n本問卷包含一系列問題，以確認該公司是否符合OpenChain合格程序。如果所有問題的回答都為“是”，則該公司符合OpenChain規範版本2.1（功能與ISO\/IEC DIS 5230:2020（e）相同）的所有要求。如果任一問題的回答為“否”，則該公司可以明確的知道需要在哪些項目加以改善以達成合規程序。",

```

Comment:
I'm not sure about the backslash in the above preamble text

Reply:
done locally, PR later.